boards/lobaro-lorabox/include/gpio_params.h

```diff
@@ -31,13 +31,11 @@ extern "C" {
  */
 static const  saul_gpio_params_t saul_gpio_params[] =
 {
-#ifdef AUTO_INIT_LED0
```

Comment:
wait stop, does this make sense? Basically you remove the usage of the MACRO here.

Reply:
see [comment](
https://github.com/RIOT-OS/RIOT/pull/10322#pullrequestreview-207482274) to

Reply:
Yes, the pin was initialized regardless of `AUTO_INIT_LED0`, it would only not show up on SAUL when not defined. As the pin is only used for the LED and is not used for other purposes it would make sense to initialize it by default without having to define it (as I understand it other boards do so because they share the pin with other peripherals).